katas/content/teleportation/entangled_pair/index.md

```diff
@@ -0,0 +1,5 @@
+**Input:** 
+Two qubits `qAlice` and `qBob`, each in $\ket{0}$ state.
+
+**Goal:**  
```

Comment:
```suggestion
**Goal:**
```
(this way the text is on the same line as the header, consistent with other tasks)